_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -22,33 +22,33 @@ acknowledgments:
 
 This rule applies to any HTML element that is [visible][] and has a [declared][] value specified via the `style` attribute for any of the following CSS properties:
 
-- [word-spacing][],
-- [letter-spacing][],
+- [word-spacing][]; or
+- [letter-spacing][]; or
 - [line-height][].
 
 ## Expectation 1
 
-For the test target's [word-spacing][] property, either:
+For the test target's [word-spacing][] property, one of the following is true:
 
-- it has no [declared][] value; or
-- its [declared][] value specified via the `style` attribute does not have the [important flag][]; or
-- its [computed][] value is at least `0.16` times its [computed][] [font-size][].
+- has no [declared][] value; or
```

Comment:
This kinda makes me think there should be three separate rules here. We're using conditional logic in the expectations to get around the issue that the applicability is too broad. What would others think about breaking this into 3 rules?

Reply:
@WilcoFiers we discussed the possibility to split this rule into 3 in this [conversation](https://github.com/act-rules/act-rules.github.io/pull/1251#discussion_r445585345).

@Jym77 and @EmmaJP lets discuss this again.

Personally I am open to it.

Reply:
I'm also open to it.
From the logical point of view, I agree that using expectations for a "or" case analysis is odd and splitting the rule would be more streamlined.
From a complexity point of view, it would be 3 super simple rules that are a bit "lost" if we look at them individually and keeping them together gives more punch to the result.

---
🤔 with the conditional list in expectations, and the weird case in each "the property has no declared value", I say I'm slightly in favour of splitting.